
This ELLIS Symposium organized by [ELLIS Health](https://ellis.eu/programs/ellis-health) and [ELLIS ML4Molecules](https://ellis.eu/programs/machine-learning-for-molecule-discovery).

## Participate
Date: Tuesday, June 4, 2024  
Time: 13:00 - 16:00 CEST  
Place: via Zoom (to request meeting details, please send an email to ml4molecules@ml.jku.at)  

## Program

### 13:00 Keynote by Prof. Francesca Grisoni
Eindhoven University of Technology, Dept. Biomedical Engineering,  
Institute for Complex Molecular Systems and Eindhoven AI Systems Institute, Eindhoven, Netherlands  
Centre for Living Technologies, Alliance TU/e, WUR, UU, UMC Utrecht, Netherlands.

**Harnessing deep learning for low-data drug discovery: lessons learned and opportunities**  
Deep learning has had an incredible impact in various fields of science and technology, such as protein structure prediction and organic reaction planning. However, it is known to be particularly effective when large-scale datasets are available. Drug discovery, on the other hand, is usually a low-data endeavour – which limits the potential of ‘out-of-the-box’ deep learning approaches. This talk will reflect on lessons learned in applying deep learning to low-data drug discovery and highlight some successful strategies to address these challenges and discover bioactive molecules.
				
### 14:00 Talk by Johannes Schimunek
ELLIS Unit Linz and LIT AI Lab, Institute for Machine Learning,  
Johannes Kepler University Linz, Austria  

**Machine Learning-guided directed protein evolution**
[TBA]

### 14:20 Coffee break

### 14:40 Talk by Dr. Carlos Oliver
Department of Machine Learning and Systems Biology  
Max Planck Institute of Biochemistry Martinsried, Germany  

**Building datasets and benchmarks for deep learning on protein structures with ProteinShake**  
We present ProteinShake, a Python software package that simplifies dataset creation and model evaluation for deep learning on protein structures. Users can create custom datasets or load an extensive set of pre-processed datasets from biological data repositories such as the Protein Data Bank (PDB) and AlphaFoldDB.
Each dataset is associated with prediction tasks and evaluation functions covering a broad array of biological challenges. A benchmark on these tasks shows that pretraining almost always improves performance, the optimal data modality (graphs, voxel grids, or point clouds) is task-dependent, and models struggle to generalize to new structures. ProteinShake makes protein structure data easily accessible and comparison among models straightforward, providing challenging benchmark settings with real-world implications.


### 15:00 Keynote by Prof. Lucy Colwell
Yusuf Hamied Department of Chemistry  
University of Cambridge, Google DeepMind researcher, United Kingdom  

**AI-generated protein function prediction with therapeutic applications**  
A central challenge in biochemistry is the prediction of the functional properties of a protein from its amino acid sequence, as it can lead to the discovery of new proteins with specific functionality and a better understanding of the functional effect of genomic mutations. Experimental and computational data enable the training and validation of ML models that predict protein function directly from sequence. This talk will present deep learning models that accurately predict functional domains within protein sequences, and large language models that generate textual descriptions of protein sequences, collectively adding millions of annotations to public databases.
Beyond sequences found in the evolutionary record, technical breakthroughs enable data on the sequence-to-function relationship to be rapidly acquired. However, the cost and latency of wet lab experiments means that we require new methods to find 'hits' (sequences that meet the function requirements of the campaign) in few experimental rounds, where each round contains a large batch of sequence designs. I will discuss model-based optimisation approaches that take advantage of sample inefficient methods to navigate sequence space and identify diverse candidates for experimental evaluation. The potential of these approaches will be illustrated through case studies demonstrating the design and experimental validation of proteins and peptides for therapeutic applications.


## Program committee
Karsten Borgwardt and Günter Klambauer  

github pages by lh
